common/src/main/java/com/wynntils/screens/maps/PoiManagementScreen.java

```diff
@@ -390,12 +385,12 @@ protected void doInit() {
 
     @Override
     public void doRender(GuiGraphics guiGraphics, int mouseX, int mouseY, float partialTick) {
-        PoseStack poseStack = guiGraphics.pose();
         super.doRender(guiGraphics, mouseX, mouseY, partialTick);
+        PoseStack poseStack = guiGraphics.pose();
         renderScrollButton(poseStack);
 
         // Uncomment when editing UI
-        //        RenderUtils.renderDebugGrid(poseStack, GRID_DIVISIONS, dividedWidth, dividedHeight);
+        // RenderUtils.renderDebugGrid(poseStack, GRID_DIVISIONS, dividedWidth, dividedHeight);
```

Comment:
Perhaps could be a method in WynntilsGridScreen, and we could remove these comments.